features/request/RequestContext.php

```diff
@@ -464,7 +464,7 @@ public function iHaveProvidedAnRporiginOf($rpOrigin)
     /**
      * @When I call mfaCreate
      */
-    public function iCallMfacreate()
+    public function iCallMfaCreate()
```

Comment:
If this function was auto-generated with the other casing, it is best to leave it that way, even though semantically this matches. You don't want PHP to change how it matches function calls (case-insensitively to sensitively) and break things.

Reply:
If it's like most of our Behat test functions, it was autogenerated. However, the matching that Behat does is purely based on the `@When` annotation in the doc. block, so changing case in the function name itself is safe here. Feel free to do so or not, as you like.